### PR TITLE
chore: build enhancements 4

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by nattallius — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en" class=" esl-scrollable-content ">
```
Comment (dshovchko) [inline docs/index.html:2]: Typo: extra spaces

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        if (!window.Promise || !window.CustomEvent) {
+            version = 'full';
+        } else if (!('getRootNode' in Element.prototype) || !window.customElements || !window.IntersectionObserver) {
+            version = 'light';
+        }
+        (version) && document.write('<script type="text/javascript" src = "./bundles/polyfill-' + version + '.js"><\/script>');
+    })();
+</script>
+
+<link type="text/css" rel="stylesheet" href = "./bundles/localdev.css">
```
Comment (dshovchko) [inline docs/index.html:21]: Typo: extra spaces

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+      <a href="mailto:astsefanovich@exadel.com">Alexey Stsefanovich</a>,
+      <a href="mailto:ymurashka@exadel.com">Julia Murashko</a>,
+      <a href="mailto:yadamska@exadel.com">Yuliya Adamskaya</a>
+    </div>
+  </div>
+</div>
+
+</footer>
+
+<script src = "./bundles/localdev.js"></script>
```
Comment (dshovchko) [inline docs/index.html:176]: Typo: extra spaces

File: docs/index.html (hunk 1 above)
Comment (dshovchko) [reply docs/index.html:2]: All pages in docs with the same issues

**Review by dshovchko — APPROVED** [review]

**Comment by dshovchko** [comment]
Gulp disappointed me very much. Glad you got rid of it!

**Review by andreybelous — APPROVED** [review]

File: pages/views/pages/esl-accordion.html [hunk 4]
```diff
@@ -13,7 +13,7 @@
         }
     </style>
     <div class="col-12">
-        <div class="accordion-group">
+        <esl-panel-stack class="accordion-group">
```
Comment (julia-murashko) [inline pages/views/pages/esl-accordion.html:16]: Mb esl-panel-group?

**Review by julia-murashko — APPROVED** [review]

File: docs/index.html (hunk 1 above)
Comment (ala-n) [reply docs/index.html:2]: Fixed, thanks!

File: pages/views/pages/esl-accordion.html (hunk 4 above)
Comment (ala-n) [reply pages/views/pages/esl-accordion.html]: oops..., Thanks, fixed

**Review by yadamskaya — APPROVED** [review]

**Pull request merged**